src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -111,6 +115,10 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
             buildImageCmd.withPull(getPull())
         }
 
+        if(getLabels()) {
```

Comment:
@bmuschko It's fine. Actually caring about all details is a good thing.

If @bklein1 would not have written this in the train he would probably had seen it by himself :)